<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ورود</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/pic/dpm_app_icon.png</normaloff>:/icon/pic/dpm_app_icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameBorder">
     <property name="styleSheet">
      <string notr="true">#frameBorder {
	border: 2px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="mainFrame">
        <property name="styleSheet">
         <string notr="true">/* ------------------------------------- */
#mainFrame {
border: 3px solid gray;
border-radius: 6px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
/* ------------------------------------- */
QLabel {
font-weight: bold;
font-size: 15px;
}
/* ------------------------------------- */
QLineEdit, QComboBox {
padding: 3px;
border-style: solid;
border: 2px solid gray;
border-radius: 2px;
font-weight: bold;
font-size: 15px;
}
/* ------------------------------------- */
#cmbUsername{

}
/* ------------------------------------- */
QPushButton{
color: #333;
border: 2px solid #555;
border-radius: 7px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 60px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
/* ------------------------------------- */

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnClose">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>74</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>بستن</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLogIn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>74</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>ورود</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="ledPassword">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::PreventContextMenu</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>123456</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblUserName">
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>نام کاربری :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pageNahadLogo">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="lblLogoNahad">
               <property name="pixmap">
                <pixmap resource="rc/rc.qrc">:/nahad-logo.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageUserLogo">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="lblLogoUser">
               <property name="pixmap">
                <pixmap resource="rc/rc.qrc">:/user-login.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblPassword">
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>کلمه عبور  :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cmbUsername">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledPassword</tabstop>
  <tabstop>btnLogIn</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="rc/rc.qrc"/>
 </resources>
 <connections/>
</ui>
